From 70ca17a97fcdae2333bca1239ce5322347d799c5 Mon Sep 17 00:00:00 2001
From: Petr Fedchenkov <giggsoff@gmail.com>
Date: Wed, 17 Aug 2022 19:37:09 +0300
Subject: [PATCH 1/1] scripts/bootstrap force build new meson

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>
---
 scripts/bootstrap.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/scripts/bootstrap.sh b/scripts/bootstrap.sh
index 8af1db858a..ba867e8f16 100755
--- a/scripts/bootstrap.sh
+++ b/scripts/bootstrap.sh
@@ -99,6 +99,9 @@ CTARGET=$TARGET_ARCH BOOTSTRAP=nobase APKBUILD=$(apkbuildname gcc) abuild -r
 # Cross build-base
 CTARGET=$TARGET_ARCH BOOTSTRAP=nobase APKBUILD=$(apkbuildname build-base) abuild -r
 
+# Cross meson because current version contains bug
+CTARGET=$TARGET_ARCH BOOTSTRAP=nobase APKBUILD=$(apkbuildname meson) abuild -r
+
 msg "Cross building base system"
 
 # Implicit dependencies for early targets
-- 
2.34.1

