From 550067fb54ccc80c5e2d01d0bc27f51bd18b4010 Mon Sep 17 00:00:00 2001
From: Petr Fedchenkov <giggsoff@gmail.com>
Date: Sun, 21 Aug 2022 16:47:39 +0300
Subject: [PATCH] main/argp-standalone add coreutils dep

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>
---
 main/argp-standalone/APKBUILD | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/main/argp-standalone/APKBUILD b/main/argp-standalone/APKBUILD
index 5799c19d1f..423ba0c8eb 100644
--- a/main/argp-standalone/APKBUILD
+++ b/main/argp-standalone/APKBUILD
@@ -7,7 +7,7 @@ pkgdesc="Hierarchial argument parsing library broken out from glibc"
 url="http://www.lysator.liu.se/~nisse/misc/"
 arch="all"
 license="GPL-2.0-or-later"
-makedepends="libtool automake autoconf"
+makedepends="libtool automake autoconf coreutils"
 source="http://www.lysator.liu.se/~nisse/misc/argp-standalone-$pkgver.tar.gz
 	001-throw-in-funcdef.patch
 	gnu89-inline.patch
-- 
2.34.1

