From 209560ba0fc183523d6f3394bf207d099a4024f9 Mon Sep 17 00:00:00 2001
From: Petr Fedchenkov <giggsoff@gmail.com>
Date: Sun, 21 Aug 2022 19:05:47 +0300
Subject: [PATCH] main/libuv add coreutils dep and fix tests in qemu

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>
---
 main/libuv/APKBUILD | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/main/libuv/APKBUILD b/main/libuv/APKBUILD
index 07992e5b62..95fb0f17c8 100644
--- a/main/libuv/APKBUILD
+++ b/main/libuv/APKBUILD
@@ -8,7 +8,7 @@ pkgdesc="Cross-platform asychronous I/O"
 url="https://libuv.org/"
 arch="all"
 license="MIT AND ISC"
-makedepends_build="automake autoconf libtool"
+makedepends_build="automake autoconf libtool coreutils"
 makedepends_host="linux-headers"
 subpackages="$pkgname-static $pkgname-dev $pkgname-dbg"
 source="https://dist.libuv.org/dist/v$pkgver/libuv-v$pkgver.tar.gz"
@@ -24,6 +24,7 @@ prepare() {
 }
 
 build() {
+    CPPFLAGS=-D__QEMU__=1 \
 	./configure \
 		--build=$CBUILD \
 		--host=$CHOST \
-- 
2.34.1

