From 4ccfd93f631e2a4d8d7d75c1cf7c77cfd0a87601 Mon Sep 17 00:00:00 2001
From: Petr Fedchenkov <giggsoff@gmail.com>
Date: Sun, 21 Aug 2022 20:33:46 +0300
Subject: [PATCH] main/dbus add coreutils dep

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>
---
 main/dbus/APKBUILD | 1 +
 1 file changed, 1 insertion(+)

diff --git a/main/dbus/APKBUILD b/main/dbus/APKBUILD
index 84a819a21f..3e43613832 100644
--- a/main/dbus/APKBUILD
+++ b/main/dbus/APKBUILD
@@ -17,6 +17,7 @@ makedepends="$depends_dev
 	libtool
 	libx11-dev
 	xmlto
+	coreutils
 	"
 checkdepends="xvfb-run"
 pkggroups="messagebus"
-- 
2.34.1

