From 6d6e706717ad1fd18dd6006160d2357b73a80b2a Mon Sep 17 00:00:00 2001
From: Petr Fedchenkov <giggsoff@gmail.com>
Date: Wed, 24 Aug 2022 13:19:35 +0300
Subject: [PATCH] main/tiff add coreutils dep

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>
---
 main/tiff/APKBUILD | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/main/tiff/APKBUILD b/main/tiff/APKBUILD
index ebfe56ece0..01ff4ea257 100644
--- a/main/tiff/APKBUILD
+++ b/main/tiff/APKBUILD
@@ -9,7 +9,7 @@ url="https://gitlab.com/libtiff/libtiff"
 arch="all"
 license="libtiff"
 depends_dev="zlib-dev libjpeg-turbo-dev libwebp-dev xz-dev zstd-dev"
-makedepends="libtool autoconf automake $depends_dev"
+makedepends="libtool autoconf automake $depends_dev coreutils"
 checkdepends="diffutils"
 subpackages="$pkgname-doc $pkgname-dev $pkgname-tools libtiffxx:_libtiffxx"
 source="https://gitlab.com/libtiff/libtiff/-/archive/v$pkgver/libtiff-v$pkgver.tar.gz
-- 
2.34.1

