From 164b2f893c315c40d806d3e2b8567bb4c734e513 Mon Sep 17 00:00:00 2001
From: Petr Fedchenkov <giggsoff@gmail.com>
Date: Mon, 29 Aug 2022 19:07:28 +0300
Subject: [PATCH] main/bluez add coreutils dep

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>
---
 main/bluez/APKBUILD | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/main/bluez/APKBUILD b/main/bluez/APKBUILD
index 3f90d683c0..745411343f 100644
--- a/main/bluez/APKBUILD
+++ b/main/bluez/APKBUILD
@@ -11,7 +11,7 @@ depends="dbus"
 replaces="udev"
 makedepends="dbus-dev libusb-compat-dev eudev-dev json-c-dev
 	libical-dev readline-dev glib-dev linux-headers
-	autoconf automake libtool ell-dev py3-docutils"
+	autoconf automake libtool ell-dev py3-docutils coreutils"
 subpackages="
 	$pkgname-dev
 	$pkgname-doc
-- 
2.34.1

