From 7f867eeb9b95b9eb06213fd42f18e56566f8ddc8 Mon Sep 17 00:00:00 2001
From: Petr Fedchenkov <giggsoff@gmail.com>
Date: Tue, 30 Aug 2022 12:48:55 +0300
Subject: [PATCH] main/numactl add coreutils dep

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>
---
 main/numactl/APKBUILD | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/main/numactl/APKBUILD b/main/numactl/APKBUILD
index ce5f9e5cf9..0064793355 100644
--- a/main/numactl/APKBUILD
+++ b/main/numactl/APKBUILD
@@ -6,13 +6,13 @@ pkgdesc="Simple NUMA policy support"
 url="https://github.com/numactl/numactl"
 arch="all"
 license="LGPL-2.1-only"
-makedepends="autoconf automake libtool linux-headers"
+makedepends="autoconf automake libtool linux-headers coreutils"
 subpackages="$pkgname-dev $pkgname-doc $pkgname-tools"
 source="numactl-$pkgver.tar.gz::https://github.com/numactl/numactl/archive/v$pkgver.tar.gz
 	musl.patch"
 
 case "$CARCH" in
-	s390x)  options="!check";; # FAIL: distance test
+	s390x|riscv64)  options="!check";; # FAIL: distance test
 	mips*)  options="!check";; # XXX: numa support not enabled on builder
 esac
 
-- 
2.34.1

